steps/rust-caching.yml

```diff
@@ -25,17 +25,16 @@ steps:
           Write-Host "##vso[task.setvariable variable=homedir]$env:HOME"
         displayName: Get Home directory (windows)
         condition: eq(variables['Agent.OS'], 'Windows_NT')
-  - ${{ if parameters.cacheRegistry }}:
```

Comment:
Not sure if this was needed/here for cleanliness or what

Reply:
This is needed for conditional inclusion of the block so it needs to stay

Reply:
Nvm I figured out why this isn’t needed